apiary.apib

```diff
@@ -795,6 +833,50 @@ This operations is available only for organization administrators.
                 "sharing": "organization"
             }
 
+### Change Bucket Sharing To Organization [POST /v2/storage/buckets/{bucket_id}/share-organization?update=1]
+Change [sharing type](https://help.keboola.com/storage/buckets/sharing/#sharing-type) of a bucket.
+Bucket is shared to organization. Any organization member can link bucket to project in organization.
+This operations is available only for organization administrators.
+
+
++ Parameters
+    + bucket_id (required) - Bucket Id
+
++ Request
+    + Headers
+
+            X-StorageApi-Token: your_token
+
++ Response 202 (application/json)
+    + Body
+
+            {
+                "sharing": "organization"
+            }
+
+### Change Bucket Sharing To Organization Project [POST /v2/storage/buckets/{bucket_id}/share-organization-project?update=1]
+Change [sharing type](https://help.keboola.com/storage/buckets/sharing/#sharing-type) of a bucket.
+Bucket is shared to organization. Any organization or organization's project member can link bucket to project.
+This operations is available only for organization administrators.
+
+
++ Parameters
+    + bucket_id (required) - Bucket Id
+
++ Request
+    + Headers
+
+            X-StorageApi-Token: your_token
+
++ Response 202 (application/json)
+    + Body
+
+            {
```

Comment:
Vracel bych tady všechno info o bucketu. Všude to tak děláme. Upravil bych pak `bucket->toApiResponse`, aby vracela i informace o sharovaní bucketů

Reply:
Aktualne sa v response vrati info o buckete: 
```JSON
{
    "uri": "https:\/\/localhost:8700\/v2\/storage\/buckets\/in.c-API-sharing",
    "id": "in.c-API-sharing",
    "name": "c-API-sharing",
    "stage": "in",
    "description": "Api tests",
    "tables": "https:\/\/localhost:8700\/v2\/storage\/buckets\/in.c-API-sharing\/tables",
    "created": "2019-09-25T17:45:11+0200",
    "lastChangeDate": null,
    "isReadOnly": false,
    "dataSizeBytes": 0,
    "rowsCount": 0,
    "isMaintenance": false,
    "backend": "snowflake",
    "sharing": "organization"
}
```

akurat to nebolo v dokumentacii **pridam to tam.**